worksheet_C/README.md

```diff
@@ -0,0 +1,17 @@
+(a) Checks if any items on the list are repeated
+
+(b) It's taking a value from the list O(n) and checking it against every other value of the list O(n^2) 
+
+(c) It still checks every item on the list against every other item
+
+(d) It’s not going over results that it has already checked
+
+(e) Yes it’s still quadratic ((n^2)-n)/2
+
+(f) O(n log n) http://bugs.java.com/bugdatabase/view_bug.do?bug_id=6804124
+
+(g) O(n) The term that is largest when n is large. O(n log n) is combined with O(n) it becomes O(n log n) + O(n)
+
+(h) The Second - Linear is faster than quadratic given a large input.
+
+(i) Lack of temporary storage space.
```

Comment:
✅ but needs more explanation (i.e. explain that sorting is likely to require temporary storage, possibly backing this up with a reference)